_includes/codeTable.html

```diff
@@ -134,7 +147,7 @@
 
 <!-- Data Table -->
 <div class="col-md-9">
-  <table class="wb-tables table table-striped table-hover" id="dataset-filter" data-wb-tables='{"order": [1, "asc"], "columnDefs": [{"targets": [4, 5], "visible": false}], "paging": false}'>
+  <table class="wb-tables table table-striped table-hover" id="dataset-filter" data-wb-tables='{"order": [1, "asc"], "columnDefs": [{"targets": [4, 5], "visible": true}], "paging": false}'>
```

Comment:
This shows all columns and there are too many to show them all.  Tags and Organization need to stay hidden and Type can also be hidden, for now.

Reply:
Tags, Organization and Type are hidden now.